### PR TITLE
Document integrated domain support (GoDaddy)

### PR DESCRIPTION
**Pull request by lokst**
Document integrated domain providers

TODOs
- [x] Rebase after https://github.com/dnsimple/dnsimple-support/pull/1058 is merged
- [x] Optimize images

### CODE REVIEW

**Comment by lokst** [comment]
Temporarily changed the base to `main` so that the CI job to generate the preview pages will run

**Comment by lokst** [comment]
Preview pages are available here: https://deploy-preview-1065--dnsimple-support.netlify.app/

**Comment by aeden** [comment]
The section should not be under "Other" it should be a section called Integrated Domain Providers and follow the same format as Integrated DNS Providers (one generic page, one page for each provider).

**Comment by lokst** [comment]
> The section should not be under "Other" it should be a section called Integrated Domain Providers and follow the same format as Integrated DNS Providers (one generic page, one page for each provider).

Thanks for the feedback @aeden, I will make the categorization changes and also have a page for each provider :thumbsup:

File: content/articles/integrated-domain-providers.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+DNSimple supports domain integration with the providers listed below.
+
+- [GoDaddy](https://www.godaddy.com)
+
+## Linking an Integrated Domain Provider to your account
+
+1. At DNSimple, go to the <label>Account</label> page, and click the <label>Integrated Providers</label> tab.
+1. Under <label>Add an integrated provider<label>, click the link button for the integrated domain provider you want to link to your DNSimple account.
+![Link an integrated domain provider](/files/account-integrated-domain-provider-link.png)
```
Comment (dallasread) [inline content/articles/integrated-domain-providers.markdown:37]: Can you use the image from production (with the GoDaddy logo that will be seen in the app)?

File: content/articles/managing-integrated-domains.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Managing Integrated Domains
+excerpt: We support managing domains at other domain providers.
+categories:
+- Domains
```
Comment (dallasread) [inline content/articles/managing-integrated-domains.markdown:5]: Should we include integrations?

```suggestion
- Domains
- Integrations
```

File: content/articles/integrated-domain-providers.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- **View domain information**: When you link an integrated domain provider to your DNSimple account, domains registered via that integrated domain provider will be imported into DNSimple and listed on the Domains page.
+- **Domain expiration notices**: Get email notifications when your integrated domains are near expiry.
+
+## Supported Integrated Domain Providers
+
+To manage integrated domains from DNSimple, you first have to link an integrated domain provider to your DNSimple account.
+
+DNSimple supports domain integration with the providers listed below.
+
+- [GoDaddy](https://www.godaddy.com)
```
Comment (dallasread) [inline content/articles/integrated-domain-providers.markdown:31]: I feel we should link/have another page for GoDaddy, similar to https://deploy-preview-1065--dnsimple-support.netlify.app/articles/integrated-dns-provider-amazon-route53/.

File: content/articles/managing-integrated-domains.markdown [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+</div>
+
+## Refreshing integrated domains {#refreshing-integrated-domains}
+
+Loading the Domains page automatically refreshes the current state of integrated domains and imports any new domains from linked integrated domain providers.
+
+
+## Managing an integrated domain {#managing-an-integrated-domain}
+
+You can manage an integrated domain by clicking on any integrated domain in the list of domains on the Domains page, to go to the domain's manage page.
```
Comment (dallasread) [inline content/articles/managing-integrated-domains.markdown:30]:
```suggestion
You can manage an integrated domain by clicking on any integrated domain in the list of domains on the Domains page, to go to the domain's Overview page.
```

File: content/articles/managing-integrated-domains.markdown [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+![Navigate to an integrated domain](/files/integrated-domain-manage.png)
+
+On the domain's manage page, you can view the details of an integrated domain, like its provider and expiry date.
+
+![Integrated domain](/files/integrated-domain.png)
+
+ On the domain's manage page, you can click on **Registration** on the sidebar, to view the domain's delegation and registration contact details.
+
+![Integrated domain registration details](/files/integrated-domain-registration-details.png)
```
Comment (dallasread) [inline content/articles/managing-integrated-domains.markdown:40]: What do you think about mentioning the DNSSEC page as well?

**Review by dallasread — COMMENTED** [review]
This looks great – left a few comments.

File: content/articles/integrated-domain-providers.markdown (hunk 1 above)
Comment (lokst) [reply content/articles/integrated-domain-providers.markdown:38]: This screenshot was taken from Sandbox, but I believe it is the same logo asset, what do you think?

File: content/articles/integrated-domain-providers.markdown (hunk 3 above)
Comment (lokst) [reply content/articles/integrated-domain-providers.markdown]: Great idea, updated to link to a separate page for GoDaddy: https://deploy-preview-1065--dnsimple-support.netlify.app/articles/integrated-domain-provider-godaddy/

File: content/articles/managing-integrated-domains.markdown (hunk 4 above)
Comment (lokst) [reply content/articles/managing-integrated-domains.markdown:31]: It looks like we are already referring to the page as a domain's manage page, in other places, so I thought of being consistent with that, what do you think?

File: content/articles/managing-integrated-domains.markdown (hunk 5 above)
Comment (lokst) [reply content/articles/managing-integrated-domains.markdown:41]: Great idea, added a reference to the DNSSEC page 🙂

**Comment by lokst** [comment]
Dear reviewers @dallasread @kojdm @itsalyse , please access the preview pages at https://github.com/dnsimple/dnsimple-support/pull/1058 🙂

File: content/articles/managing-integrated-domains.markdown (hunk 2 above)
Comment (lokst) [reply content/articles/managing-integrated-domains.markdown:5]: Implemented :+1:

File: content/articles/integrated-domain-providers.markdown (hunk 1 above)
Comment (dallasread) [reply content/articles/integrated-domain-providers.markdown:38]: I may have commented in the wrong place. This is the one I was referring to:

https://github.com/dnsimple/dnsimple-support/pull/1065/files#diff-76c0da71781a76b5a2dcaab7777b2e6f8c301710f07209156dd5e42476d04f91

File: content/articles/managing-integrated-domains.markdown (hunk 4 above)
Comment (dallasread) [reply content/articles/managing-integrated-domains.markdown:31]: 🤷‍♂️ Sounds fine to me!

**Review by dallasread — APPROVED** [review]
Looks great! Thanks for doing this!

File: content/articles/integrated-domain-providers.markdown (hunk 1 above)
Comment (lokst) [reply content/articles/integrated-domain-providers.markdown:38]: Ah I see! Fixed here: https://github.com/dnsimple/dnsimple-support/pull/1065/commits/866426689b7ed204b9dd86a8591d5d45ba4446e4 🎉

**Review by kojdm — APPROVED** [review]
LGTM! Thanks 💯

**Pull request merged**